readme.md

```diff
@@ -159,6 +159,7 @@ GitHub Enterprise is also supported. More info in the options.
 - [Dropdown menus are automatically closed when they’re no longer visible.](https://user-images.githubusercontent.com/1402241/37022353-531c676e-2155-11e8-96cc-80d934bb22e0.gif)
 - [Permalinked comments are highlighted.](https://user-images.githubusercontent.com/1402241/37349492-226bd37a-2709-11e8-8087-d9686b330240.png)
 - [Inactive deployments in PR timelines are hidden.](https://github.com/sindresorhus/refined-github/issues/1144)
+- Link to quickly swap branches when doing a branch compare.
```

Comment:
This should probably be in the _actions_ sections with wording like "Swap branches in the branch compare view"

Reply:
I couldn't think of good wording, thanks.